app/src/main/java/com/github/geohunt/app/ui/components/CreateNewChallenge.kt

```diff
@@ -37,17 +37,6 @@ import java.io.File
 import java.text.SimpleDateFormat
 import java.util.*
 
-
```

Comment:
Made this a common util function so it can be used for both `CreateNewChallenge.kt` and `ClaimChallenge.kt`